service/host_test.go

```diff
@@ -0,0 +1,46 @@
+package service
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/evergreen-ci/evergreen/model/user"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestModifyHostStatus(t *testing.T) {
```

Comment:
Could you rewrite this test with testify rather than goconvey? We're using testify for new tests because it's clearer to read and it's easier to understand the execution semantics